client/src/components/footer/Footer.scss

```diff
@@ -0,0 +1,127 @@
+@import '../../scssStyles/_mixins.scss';
+
+.footer-container {
```

Comment:
Can you add some horizontal padding? Maybe just for desktop
So stuff is pushed away from the side